test/integration/targets/inventory_docker_machine/playbooks/test_inventory_1.yml

```diff
@@ -0,0 +1,54 @@
+- hosts: 127.0.0.1
+  gather_facts: no
+  vars:
+    dm_machine_name: "{{ lookup('env', 'DM_MACHINE_NAME') }}"
+  tasks:
+  - name: sanity check Docker Machine output
+    vars:
+      dm_ls_format: !unsafe '{{.Name}} | {{.DriverName}} | {{.State}} | {{.URL}} | {{.Error}}'
+      success_regex: "^{{ dm_machine_name }} | digitalocean | Running | tcp://.+ |$"
```

Comment:
I would remove every dependence on `digitalocean` from the tests (except from machine startup maybe), so that one can also test against other drivers without having to adjust a lot.

Reply:
Fixed the case you identified, however my code still uses Digital Ocean as the Digital Machine driver, not Virtual Box, as I wasn't yet able to get VB working in my setup.

Reply:
@felixfontein: Should I update the PR to use VirtualBox instead of DigitalOcean?